jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ui/UiUtils.kt

```diff
@@ -299,6 +302,13 @@ fun Cell.contextualHelp(description: String): CellBuilder<JBLabel> {
     return component(l)
 }
 
+fun CellBuilder<EnvironmentVariablesTextField>.withBinding(binding: PropertyBinding<Map<String, String>>) =
+    this.withBinding(
+        componentGet = { component -> component.envVars },
+        componentSet = { component, value -> component.envVars = value },
+        binding
+    )
```

Comment:
Should we just embrace this as UI DSL completely and make a `Row.keyValueTextField(property: KMutableProperty<Map<String,String>>)`?

Reply:
added, but not used here since there are ui issues with async values being populated after panel initialization and still need to figure out how to make that work in a nice way